CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # 13.1.0 Release notes
 
 ### Enhancements
+* Allows ignoring nested property changes when subscribing for notifications by passing an empty `key_path_array` to `add_notification_callback`. ([#6122](https://github.com/realm/realm-core/pull/6122)).
```

Comment:
Wrong changelog section.

This also is a breaking change as the meaning of an empty array has changed and you now need to pass nullopt for the old meaning.